log/model_formats.go

```diff
@@ -0,0 +1,57 @@
+package log
+
+import (
+	"encoding/base64"
+	"time"
+)
+
+// Renderer is an interface to deal with typing problems of just having an interface{} as the acceptable parameters
+// Go-kit logging has a failure case if you attempt to throw any values into it.
+// This is a way to guard our developers from having to worry about error cases of the lower logging framework.
+type Renderer interface {
+	getValue() interface{}
+}
+
+type any struct {
+	value interface{}
+}
+
+func (a *any) getValue() interface{} {
+	return a.value
+}
+
+func String(s string) Renderer {
+	return &any{s}
+}
+
+func Int(i int) Renderer {
+	return &any{i}
+}
+
+func Float64(f float64) Renderer {
+	return &any{f}
+}
+
+func Bool(b bool) Renderer {
+	return &any{b}
+}
+
+func Duration(d time.Duration) Renderer {
```

Comment:
```suggestion
func TimeDuration(d time.Duration) Renderer {
```

This kind of makes it nice for autocompletion purposes.